### ISSUE TITLE
Add ability to configure uMatrix in "Open as Window" mode

### ISSUE THREAD

**Issue by andrewpmartinez** [gorhill/uMatrix#596]
When using sites in "Open as Window" mode in Chrome there is no way to manage uMatrix as the extensions area is hidden. This causes issues when a site you have previously visited begins to use resources from domains that have not been allowed.

The work around is to open the site in a regular Chrome window and configure the site that way.

Configuring "Open as Window":
1) Open Chrome to a site
2) Click the **menu** button (top right)
3) Click **More Tools**
4) Click **Add to desktop...**
5) Give the site a name and ensure **Open as Window** is checked
6) Go to your **desktop** and use the new shortcut placed there (or right click it and add it to your task bar)
7) Note that uMatrix is not configurable

**Comment by gorhill** [comment]
Where is this "Open as Window" option?

**Comment by JoelAzaria** [comment]
I agree that this should be easier.  As a workaround for now, try pressing ctrl+U which is the shortcut for umatrix.  For some reason this doesn't work for me on *some* windows but most of the time it does.
